test/integration/integration_test.go

```diff
@@ -402,6 +402,10 @@ func (i *integrationTest) exerciseInjectionBuild(tag, imageName string, injectio
 			t.Errorf("injectionList.Set() failed with error %s\n", err)
 		}
 	}
+	// Don't truncate the first injected item.
```

Comment:
why?

Reply:
I decided to keep the behavior of `Set` in place, because I couldn't add a third option to the injection string without breaking Windows directory paths.

Setting `Truncated = false` ensures that we always exercise the "keep" option during the integration test.